### ISSUE TITLE
ASP.NET Dore docs for performance are broken

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-docs#8532]
https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/performance/

Missing the `UseTracingMiddleware` instructions.

This way users will not get any transactions.
Must have regressed at some point

**Comment by kanadaj** [comment]
Correction: `UseSentryTracing`

**Comment by jamescrosswell** [comment]
@bruno-garcia not regressed... updated to reflect this change in 3.40.0:
- https://github.com/getsentry/sentry-dotnet/pull/2602

**Comment by bruno-garcia** [comment]
Context about this change and side effects on Discord: https://discord.com/channels/621778831602221064/1175852280625500281/1175901670702600323

@jamescrosswell there's no reference of this at all on the docs: https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/performance/

We could expand on this, for users coming back after using the middleware and new users, that wonder what setting the sample rate does

**Comment by bruno-garcia** [comment]
Verify in the docs tells ppl to go to the Custom Transaction:

> Verify
Test out tracingThe process of logging the events that took place during a request, often across multiple services. by starting and finishing a transaction, which you must do so transactions can be sent to Sentry. Learn how in our [Custom Instrumentation](https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/performance/instrumentation/custom-instrumentation/) content.

> While you're testing, set TracesSampleRate to 1.0, as that ensures that every transaction will be sent to Sentry.

> Once testing is complete, you may want to set a lower TracesSampleRate value, or switch to using TracesSampler to selectively sample and filter your transactions, based on contextual data.

**Comment by jamescrosswell** [comment]
> https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/performance/
> 
> We could expand on this, for users coming back after using the middleware and new users, that wonder what setting the sample rate does

Do you mean like some docs saying that people no longer need to call `UseSentryTracing`? Maybe we add something after completing https://github.com/getsentry/sentry-dotnet/issues/2865 (explaining in which circumstances they would/wouldn't need to call this themselves manually)?

**Comment by bruno-garcia** [comment]
> Do you mean like some docs saying that people no longer need to call UseSentryTracing? Maybe we add something after completing https://github.com/getsentry/sentry-dotnet/issues/2865 (explaining in which circumstances they would/wouldn't need to call this themselves manually)?

Yeah we usually write stuff on a migration document (https://docs.sentry.io/platforms/dotnet/migration/)
A section on the aspnet core docs for performance clarifying how this works (calling out "From version XYZ you no longer need to call UseTracing because...") that's a good opportunity to explain how it works. Where we added it, and edge cases, etc.
